portfolio/src/main/webapp/index.html

```diff
@@ -5,7 +5,7 @@
       <title>My Portfolio</title>
       <link rel="stylesheet" href="style.css">
   </head>
-  <body onload="getHelloName()">
+  <body onload="load()">
```

Comment:
Sorry, I don't remember, was this previously changed in javascript already? I don't see a change from `getHelloName` to `load` in this PR.

Reply:
It was changed in an earlier PR but I forgot to change the function name in the html file.